docs/docs/dev/auth_flow.md

```diff
@@ -1,36 +1,7 @@
-# Authorization and authentication flow
+# Authorization/authentication flow
 
 Bacalhau authenticates and authorizes users in a multi-step flow.
 
-## Requirements
```

Comment:
What is the motivation for changing this file? A lot of this content was helpful, why is it being removed?